### PR TITLE
fix for replace decrecated btc rpc call generate

### PR DESCRIPTION
**Pull request by crove-iov**
Replace generate rpc call deprecated in latest bitcoind version

### CODE REVIEW

File: lib/mining-utils.js [hunk 1]
```diff
@@ -9,7 +9,14 @@ const createKeccakHash = require('keccak');
 const CPV_JUMP_FACTOR = 7;
 const CPV_JUMP = 64;
 const MAX_NUMBER_OF_UNCLES = 3;
-
+const host = config.rskd.url + ":" + config.rskd.rpcport;
```
Comment (lucasvuotto) [inline lib/mining-utils.js]: `defaultContext`'s `Host:` header gets populated with this, and then used to call a Bitcoin's RPC, not RSK one. You should either use `config.bitcoind.*` in here, or create a `defaultBTCContext` with `config.bitcoind.*` and use that one in the added RPC calls.

**Review by lucasvuotto — CHANGES_REQUESTED** [review]

File: lib/mining-utils.js (hunk 1 above)
Comment (crove-iov) [reply lib/mining-utils.js]: fixed, thanks

File: .circleci/config.yml [hunk 2]
```diff
@@ -72,6 +70,7 @@ jobs:
               echo "Waiting for RskJ..."
               sleep 1
             done
+            node generateBtcBlocks.js
```
Comment (lucasvuotto) [inline .circleci/config.yml]: I think I'd prefer running this right after `npm install`, but I'm fine either way.

**Review by lucasvuotto — APPROVED** [review]
FTR, `.circleci/config.yml` lines 16 and 26 include `"-deprecatedrpc=generate"` which isn't needed now. Not a stopper, but would be great if you could remove then before merging.

**Review by core-baguira — APPROVED** [review]

**Pull request merged**